portal/src/main/webapp/js/src/oncoprint/studyview.js

```diff
@@ -31,9 +115,47 @@ $(document).ready(function() {
 			})();
 			$('#outer_loader_img').hide();
 			$('#oncoprint #everything').show();
+                        
+                        var gene_data = response.toJSON();
+                        //get to process data from gene panel
+                        if(genepanelValues.genepaneldata.length>0)
+                        {
```

Comment:
maybe let's just stick with same line bracket for this file since I was already doing it that way?